src/tribble/transform.py

```diff
@@ -28,14 +29,20 @@
 
 
 def transform(df: pandas.DataFrame) -> pandas.DataFrame:
-    df = _apply_transform_list(df, TRANSFORMERS)
-    return df.groupby('uuid').apply(_group_transform_df)
+    print("Applying " + str(len(TRANSFORMERS)) + " regular transformers:")
+    df = _apply_transform_list(df, TRANSFORMERS, show_progress=True)
+    uuid_grouped = df.groupby('uuid')
+    tqdm.pandas()
```

Comment:
Very cool.

Reply:
Why is this call to `tqdm.pandas()` only _after_ we've already applied all the non-group transforms, but _before_ we apply the group transforms?

Reply:
The reason I got away with it like that, is that this statement is only needed to enhance the pandas DataFrame class with the progress_apply procedure. In _apply_transform_list we only use tqdm directly (without pandas). But maybe it would be a good convention to put it into the global scope right after the imports (because that's what it tinkers with). What do you think?

Reply:
Ah, gotcha. Yeah, I think it'd be better to put it in the global scope, but agreed that it doesn't make a difference ATM.